arch/arm/core/fatal.c

```diff
@@ -87,3 +87,9 @@ void _do_kernel_oops(const NANO_ESF *esf)
 {
 	_NanoFatalErrorHandler(esf->r0, esf);
 }
+
+FUNC_NORETURN void _arch_syscall_oops(void *ssf_ptr)
+{
+	_do_kernel_oops((const NANO_ESF *)ssf_ptr);
```

Comment:
just wanted to double-check, is ssf_ptr really a NANO_ESF?
on x86 it wasn't and I had to copy stuff.

Reply:
I'll double check.

Reply:
i still have to fix this.  I'll push another change tomorrow to address the stack frame.  I think I'll have to copy the svc stack frame or reconstruct it from the pieces I have on hand (like the x86 method)